deluge/ui/web/server.py

```diff
@@ -567,7 +576,7 @@ def __init__(self, options=None):
         self.web_utils = WebUtils()
 
         self.auth = Auth(self.config)
-        self.standalone = True
+        self.standalone = standalone
```

Comment:
Just wonder if this should be 'shared_reactor' instead of 'standalone'. Not keen on the standalone in code :)